libs/sdk-ui-tests-e2e/package.json

```diff
@@ -160,6 +160,7 @@
         "typescript": "4.0.2",
         "webpack": "^5.58.0",
         "webpack-cli": "^4.9.2",
-        "webpack-dev-server": "^4.9.1"
+        "webpack-dev-server": "^4.9.1",
+        "@4tw/cypress-drag-drop": "^1.4.0"
```

Comment:
do we need to add more dependency? without this dependency, dnd still works well

Reply:
no need, reverted